specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2020-04-01-preview/EventGrid.json

```diff
@@ -7209,6 +7229,11 @@
           "description": "Source associated with this partner topic. This represents a unique partner resource.",
           "type": "string"
         },
+        "expirationTimeIfNotActivatedUtc": {
```

Comment:
>expirationTimeIfNotActivatedUtc [](start = 9, length = 31)

same comment as above.

Reply:
Thanks for the suggestion. We need to keep it as is. Please refer to reason above.